worldedit-core/src/main/java/com/boydti/fawe/config/Settings.java

```diff
@@ -393,6 +393,12 @@
             "This will increase time taken slightly."
         })
         public boolean ALLOW_TICK_EXISTING = true;
+        @Comment({
+            "Do not wait for a chunk's history to save before sending it",
+            " - Any following history commands (undo/redo) will still need to wait for history to be saved",
+            " - Requires combine_stages = true"
+        })
+        public boolean SEND_BEFORE_HISTORY = true;
```

Comment:
Maybe we can rephrase that a bit? I don't think every end user knows what it means.

Reply:
"Undo/redo commands will wait until the history has been written to disk before executing" ?

Reply:
That sounds better.